### ISSUE TITLE
Bug: Goodreads icon doesn't seem to work

### ISSUE THREAD

**Issue by sejalv** [FortAwesome/Font-Awesome#20059]
### Bug description

Hi!
The Goodreads icon doesn't seem to display.
Tried the following:
* `<i class="fa-brands fa-goodreads" aria-hidden="true"></i>`
* `<i class="fa fa-goodreads" aria-hidden="true"></i>`
* `<i class="fa-solid fa-goodreads" aria-hidden="true"></i>`

The same applies for `fa-goodreads-g`

### Reproducible test case

https://fontawesome.com/icons/goodreads?f=brands&s=solid

### Screenshots

![Screenshot 2023-12-24 at 16 46 29](https://github.com/FortAwesome/Font-Awesome/assets/9307691/dc00ee7b-7aa7-47eb-a663-a8e8a700e0ab)
![Screenshot 2023-12-24 at 16 47 55](https://github.com/FortAwesome/Font-Awesome/assets/9307691/5ff5b16b-d06c-495b-9a87-6222ccf9f562)
![Screenshot 2023-12-24 at 16 46 41](https://github.com/FortAwesome/Font-Awesome/assets/9307691/8d7d9fb0-275b-4bcf-b2d6-a2dafad81413)
![Screenshot 2023-12-24 at 16 48 22](https://github.com/FortAwesome/Font-Awesome/assets/9307691/46529a26-08c6-461e-a807-caee5e9d2c75)

### Font Awesome version

v5.0.0

### Serving

Self-hosted

### Implementation

Other (as specified in the bug description)

### Browser and Operating System

- Chrome on macOS (10.15)

### Web bug report checklist

- [ ] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

> * `<i class="fa-brands fa-goodreads" aria-hidden="true"></i>`
> * `<i class="fa fa-goodreads" aria-hidden="true"></i>`
> * `<i class="fa-solid fa-goodreads" aria-hidden="true"></i>`

None of the above is correct for Font Awesome 5, please refer to v5 docs: https://fontawesome.com/v5/docs/web/reference-icons/

The correct markup is:

```
<i class="fab fa-goodreads"></i>
```

as per https://fontawesome.com/v5/icons/goodreads?f=brands&s=solid

Demo: https://jsfiddle.net/tagliala/cq9rdny4/15/

Hope it helps, closing here.

Feel free to comment

**Comment by sejalv** [comment]
Hi, thanks for linking in the [page for v5.15.4](https://fontawesome.com/v5/icons/goodreads?f=brands&s=solid) which has a different usage (`fab`) from the [latest page/version](https://fontawesome.com/icons/goodreads?f=brands&s=solid) (`fa-brands`).

Unfortunately for me neither works, even though I'm able to use the same in the fiddle you shared. I'll check what version of `font-awesome` I'm using to find the appropriate notation.

Thanks,
SV.

**Comment by tagliala** [comment]
Please also check for third-parties, they may install older versions of FA

If you can provide a reproducible test case I can take a look

**Comment by sejalv** [comment]
Thank you for the hint! It is indeed third-party that I'm rendering on my base-level html page: https://www.bootstrapcdn.com/fontawesome/. `bootstrapcdn` offers v6, but upgrading the link to `https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.2/css/fontawesome.min.css` hasn't worked for me yet. I'll look for other options.

`base.html`
```
<!DOCTYPE html>
<html lang="en">
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
body>
  <div id="barba-wrapper">
    <div class="barba-container">
      {{ content }}
    </div>
  </div>
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
</body>
</html>
```

**Comment by tagliala** [comment]
```
https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css
```

goodreads is not available in 4.7

Please take a look at https://fontawesome.com/docs/web/setup/upgrade/
